packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryConfirmKey/index.tsx

```diff
@@ -308,7 +309,10 @@ const AccountRecoveryConfirmKey = ({
           to={`/complete_reset_password${location.search}`}
           className="link-blue text-sm"
           id="lost-recovery-key"
-          state={{ lostRecoveryKey: true }}
+          state={{
+            lostRecoveryKey: true,
+            accountResetToken: fetchedResetToken,
```

Comment:
I tried to add a test to check this link's `state`, and failed since I could only see an HTML element. There's potentially some weird ways to mock this if we really wanted but I feel like the test coverage is sufficient.